.gitignore

```diff
@@ -6,7 +6,6 @@ ignored-assets
 main.wasm
 .parcel-cache
 .vscode/*.log
-go.sum
```

Comment:
A few go.sum files were checked in before ignored. Some TinyGo examples have a go.sum file while others don't.  I figured it's best to leave them in.